client/src/app/tabs/dmn/DmnEditor.js

```diff
@@ -401,18 +401,22 @@ export class DmnEditor extends CachedComponent {
     let editMenu;
 
     if (activeView.type === 'drd') {
+
+      const canvasFocused = activeViewer.get('canvas').isFocused();
+
       assign(newState, {
         align: selectionLength > 1,
-        defaultCopyCutPaste: inputActive,
-        defaultUndoRedo: inputActive,
+        canvasFocused,
+        defaultCopyCutPaste: !canvasFocused,
+        defaultUndoRedo: !canvasFocused,
         distribute: selectionLength > 2,
-        editLabel: !inputActive && !!selectionLength,
-        find: !inputActive,
-        lassoTool: !inputActive,
-        moveCanvas: !inputActive,
-        moveSelection: !inputActive && !!selectionLength,
-        removeSelected: inputActive || !!selectionLength,
-        selectAll: true,
+        editLabel: canvasFocused && !!selectionLength,
+        find: canvasFocused,
+        lassoTool: canvasFocused,
+        moveCanvas: canvasFocused,
+        moveSelection: canvasFocused && !!selectionLength,
+        removeSelected: !canvasFocused || !!selectionLength,
+        selectAll: canvasFocused,
```

Comment:
Since we didn't release cmmn-js, why would the canvas be updated? I'd say let's not touch cmmn part at all.